Core/src/main/java/net/dv8tion/jda/player/source/RemoteSource.java

```diff
@@ -232,7 +236,7 @@ public AudioStream asStream()
         ytdlLaunchArgs.add("--");   //Url separator. Deals with YT ids that start with --
         ytdlLaunchArgs.add(url);    //specifies the URL to download.
 
-        return new RemoteStream(ytdlLaunchArgs, ffmpegLaunchArgs);
+        return new RemoteStream(ytdlLaunchArgs, ffmpegLaunchArgs, this.guildId);
```

Comment:
While using `this` is not needed (in this case) it's always good to specify which instance we are working with.